src/ImageRunModal.jsx

```diff
@@ -731,6 +756,12 @@ export class ImageRunModal extends React.Component {
                         </FormGroup>
                         }
 
+                        {dialogValues.entrypoint &&
+                        <FormGroup fieldId='run-image-dialog-entrypoint' hasNoPaddingTop label={_("Entrypoint")}>
+                            {dialogValues.entrypoint}
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1155-20230104-111228-5fe35f89-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)